nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2074,7 +2074,25 @@ The Provenance Repository contains the information related to Data Provenance. T
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository and should only be changed with caution. To store provenance events in memory instead of on disk (at the risk of data loss in the event of power/machine failure), set this property to org.apache.nifi.provenance.VolatileProvenanceRepository.
+|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository.
+Two additional repositories are available as and should only be changed with caution.
+To store provenance events in memory instead of on disk (at the risk of data loss in the event of power/machine failure),
+set this property to org.apache.nifi.provenance.VolatileProvenanceRepository. This leaves a configurable number of Provenance Events in the Java heap, so the number
+of events that can be retained is very limited. It has been used essentially as a no-op repository and is not recommended.
```

Comment:
I think all we need to say here is that _VolatileProvenanceRepository_ stores events in memory and is configurable. However, once event buffer exceeds, events are evicted and essentially lost, and that is expected. There is no *risk* here, just different expectations.